components/autocomplete/overview.md

```diff
@@ -86,6 +86,7 @@ The Blazor AutoComplete provides various parameters that allow you to configure
 | Parameter    | Type  | Description |
 | ----------- | ----------- | -------|
 | `Data` | `IEnumerable<TItem>` | allows you to provide the data source. Required.
+| `DebounceDelay` | `int` <br/> 150 | Time in milliseconds between the last typed symbol and the firing of the internal oninput event. Applies to the filtering as well since it occurs upon user input.
```

Comment:
When you say "Applies to the filtering AS WELL", what else does it apply to?

```suggestion
| `DebounceDelay` | `int` <br/> (`150`) | The milliseconds between the last typed symbol and the firing of the internal oninput event. Applies to the filtering as well, since it occurs upon user input.
```

Reply:
Will reword it to cover scenarios when filtering is not enabled as well.